src/display.asm

```diff
@@ -86,3 +86,17 @@ print_string:
 .done:
   pop ax
   ret
+
+render_controlls:
+  push dx
+  mov dx, 0x0000
+  mov si, left_string
```

Comment:
My current calling convention is to push and pop all registers that are manipulated by the callee. This function should therefore also push and pop the `si` register.

Reply:
Oh, I see.
Therefore, I will fix print_window subroutine in the same way.